marketplace/wpp_templates/models.py

```diff
@@ -0,0 +1,116 @@
+import uuid
+
+from django.db import models
+
+from rest_framework.exceptions import ValidationError
+
+
+class TemplateMessage(models.Model):
+    uuid = models.UUIDField(default=uuid.uuid4, editable=False, unique=True)
+    name = models.CharField(max_length=60)
+
+    CATEGORY_CHOICES = (
+        ("ACCOUNT_UPDATE", ""),
+        ("PAYMENT_UPDATE", ""),
+        ("PERSONAL_FINANCE_UPDATE", ""),
+        ("SHIPPING_UPDATE", ""),
+        ("RESERVATION_UPDATE", ""),
+        ("ISSUE_RESOLUTION", ""),
+        ("APPOINTMENT_UPDATE", ""),
+        ("TRANSPORTATION_UPDATE", ""),
+        ("TICKET_UPDATE", ""),
+        ("ALERT_UPDATE", ""),
+        ("AUTO_REPLY", ""),
+        ("TRANSACTIONAL", ""),
+        ("MARKETING", ""),
+        ("OTP", ""),
+    )
+
+    category = models.CharField(max_length=20, choices=CATEGORY_CHOICES)
+    created_on = models.DateField()
+
+    TEMPLATE_TYPES_CHOICES = (
+        ("MEDIA", ""),
+        ("INTERACTIVE", ""),
+        ("TEXT", ""),
+    )
+
+    template_type = models.CharField(max_length=10, choices=TEMPLATE_TYPES_CHOICES)
+    namespace = models.CharField(max_length=60)
+
+    def verify_namespace():
+        pass
+
+
+class TemplateTranslation(models.Model):
+    uuid = models.UUIDField(default=uuid.uuid4, editable=False, unique=True)
+
+    STATUS_CHOICES = (
+        ("APPROVED", ""),
+        ("IN_APPEAL", ""),
+        ("PENDING", ""),
+        ("REJECTED", ""),
+        ("PENDING_DELETION", ""),
+        ("DELETED", ""),
+        ("DISABLED", ""),
+        ("LOCKED", ""),
+    )
+
+    template = models.ForeignKey(TemplateMessage, on_delete=models.PROTECT, related_name="template_translations")
```

Comment:
The related_name can just be `translations` like: https://github.com/Ilhasoft/rapidpro/blob/main/temba/templates/models.py#L69